cypress/integration/group2/ToolsSearch.ts

```diff
@@ -1,54 +0,0 @@
-/*
- *    Copyright 2018 OICR
- *
- *    Licensed under the Apache License, Version 2.0 (the "License");
- *    you may not use this file except in compliance with the License.
- *    You may obtain a copy of the License at
- *
- *        http://www.apache.org/licenses/LICENSE-2.0
- *
- *    Unless required by applicable law or agreed to in writing, software
- *    distributed under the License is distributed on an "AS IS" BASIS,
- *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *    See the License for the specific language governing permissions and
- *    limitations under the License.
- */
-import { resetDB, setTokenUserViewPort } from '../../support/commands';
-
-describe('Dockstore tool list page', () => {
-  resetDB();
-  setTokenUserViewPort();
-  describe('Select a tool', () => {
-    it('Should be able to go to the tools search page', () => {
-      cy.visit('/search-containers');
```

Comment:
Sorry, on second thought. 
Let's leave dockstore.org/search-containers and dockstore.org/tools working (as with their workflow equivalents). Its good to remove them from the navigation bar and the prominent browse buttons on the homepage but they can be useful for debugging if elastic search is buggy/down. 

Maybe bury the links under the homepage in "Adding to Dockstore" or even deeper in a sitemap and leave the tests for the pages themselves. 

FYI, I think the "search tool" button on the current homepage isn't working quite right and doesn't select the entry type when clicking through, please confirm and open an issue if so.

Reply:
So you want me to readd the pages which have the tables for tools and workflows? Do you want me to remove the redirects too?

Reply:
Also I checked and the search tool works.